rpc/errors.go

```diff
@@ -18,6 +18,22 @@ package rpc
 
 import "fmt"
 
+const (
+	// Engine API errors
+	SERVER_ERROR         = -32000
+	INVALID_REQUEST      = -32600
+	METHOD_NOT_FOUND     = -32601
+	INVALID_PARAMS_ERROR = -32602
+	INTERNAL_ERROR       = -32603
+	INVALID_JSON         = -32700
+
+	UNKNOWN_PAYLOAD            = -38001
+	INVALID_FORKCHOICE_STATE   = -38002
+	INVALID_PAYLOAD_ATTRIBUTES = -38003
+	TOO_LARGE_REQUEST          = -38004
+	UNSUPPORTED_FORK_ERROR     = -38005
+)
+
```

Comment:
Right, but I am just not fun of code duplicating thing. It just doesn't lead to a good

Reply:
I think you are partially right, I understand what you mean, but I think it's batter to keep error related thing in one place, it's just how I would do it. You can call it design choice :). Anyway this is not a huge performance boost, so we can leave as is

Reply:
I see them as error codes. If any error code in the future will change you will have to go all over the code to change them manually instead of changing a value in a single place. It probably makes sense to extract errors from rpc as well in the future in case we will do major refactoring, but not now. we are leaving it as is, right?